crates/goose/src/providers/oauth.rs

```diff
@@ -99,7 +99,7 @@ async fn get_workspace_endpoints(host: &str) -> Result<OidcEndpoints> {
     if !resp.status().is_success() {
         return Err(anyhow::anyhow!(
             "Failed to get OIDC configuration from {}",
-            oidc_url.to_string()
+            oidc_url
```

Comment:
unrelated change?

Reply:
incidental anyhow bump from 1.0.99 → 1.0.100 makes it possible for clippy to figure out the to_string is redundant.